### ISSUE TITLE
Next and Previous posts

### ISSUE THREAD

**Issue by janiosarmento** [mixu/ghost-render#2]
It seems this project is far behind the official Ghost.

I'm trying to convert my blog, but the features `{{#next_post}}` and `{{#prev_post}}` do not work, and they are very important to me (otherwise the blog won't have any kind of navigation).

Any chance of updating the renderer?

(Unfortunately I'm just a user, I don't know how I could fix this by myself. Sorry.)

**Comment by mixu** [comment]
yeah - the current version was forked around ~0.5.3 so helpers added after that would need to be backported from this list: http://themes.ghost.org/docs/changelog 

As of right now I don't really have the time to do it, but I'm hoping to get to it. In the meanwhile I am happy to accept pull requests. Basically, they should port code from this folder: https://github.com/TryGhost/Ghost/tree/master/core/server/helpers into this file: https://github.com/mixu/ghost-render/blob/master/lib/ghost/helpers/helpers.js

the overall logic is here: https://github.com/mixu/ghost-render/blob/master/bin/ghost-render#L141 which may need to be adjusted so that the required metadata is available

(just putting this out there if someone else has the time)

**Comment by janiosarmento** [comment]
I’ll take a look. I don’t think I’ll be able to do it, but perhaps it is a
good starting point to me. Thanks.
